diff --git a/Tables/dbo.Employees.sql b/Tables/dbo.Employees.sql
index f88e7cc..6cf16a9 100644
--- a/Tables/dbo.Employees.sql
+++ b/Tables/dbo.Employees.sql
@@ -17,8 +17,7 @@
 [Photo] [image] NULL,
 [Notes] [ntext] COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [ReportsTo] [int] NULL,
-[PhotoPath] [nvarchar] (255) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
-[RegionID] [int] NULL
+[PhotoPath] [nvarchar] (255) COLLATE SQL_Latin1_General_CP1_CI_AS NULL
 ) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
 GO
 ALTER TABLE [dbo].[Employees] WITH NOCHECK ADD CONSTRAINT [CK_Birthdate] CHECK (([BirthDate]<getdate()))
diff --git a/Tables/dbo.Region.sql b/Tables/dbo.Region.sql
new file mode 100644
index 0000000..67ecdd3
--- /dev/null
+++ b/Tables/dbo.Region.sql
@@ -0,0 +1,8 @@
+CREATE TABLE [dbo].[Region]
+(
+[RegionID] [int] NOT NULL,
+[RegionDescription] [nchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[Region] ADD CONSTRAINT [PK_Region] PRIMARY KEY NONCLUSTERED  ([RegionID]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.Territories.sql b/Tables/dbo.Territories.sql
index b894b82..f6de4a8 100644
--- a/Tables/dbo.Territories.sql
+++ b/Tables/dbo.Territories.sql
@@ -2,8 +2,10 @@
 (
 [TerritoryID] [nvarchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
 [TerritoryDescription] [nchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
-[Region] [nchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL
+[RegionID] [int] NOT NULL
 ) ON [PRIMARY]
 GO
 ALTER TABLE [dbo].[Territories] ADD CONSTRAINT [PK_Territories] PRIMARY KEY NONCLUSTERED  ([TerritoryID]) ON [PRIMARY]
 GO
+ALTER TABLE [dbo].[Territories] ADD CONSTRAINT [FK_Territories_Region] FOREIGN KEY ([RegionID]) REFERENCES [dbo].[Region] ([RegionID])
+GO
